scripts/components/SongCards.js

```diff
@@ -101,7 +101,7 @@ class SongCards extends Component {
 
     renderSongs(start, end) {
         const chunk = 5;
-        const {authed, dispatch, playlist, playlists, playingSongId, songs, users} = this.props;
+        const {authed, dispatch, playlist, playlists, playingSongId, songs, users, toggleStats} = this.props;
```

Comment:
Thanks for going deeper on this explanation.  Would have just taken `mapStateToProps` as a standard react best-practice, if you didn't mention anything.

Anyways, prop data made sense to me with your use of it :+1: